client/state/login/utils.js

```diff
@@ -66,22 +67,19 @@ export function getErrorFromHTTPError( httpError ) {
 		if ( code in errorFields ) {
 			field = errorFields[ code ];
 		} else if ( code === 'compromisable_account' ) {
-			const url = localizeUrl( 'https://wordpress.com/wp-login.php?action=lostpassword' );
 			return {
 				code,
 				message: (
 					<p>
 						{ translate(
 							'Your account has been blocked as a security precaution. To continue, you must {{a}}reset your password{{/a}}.',
-							{ components: { a: <a href={ url } rel="external" /> } }
+							{ components: { a: <a href={ lostPassword() } rel="external" /> } }
```

Comment:
Although this was like that before, current locale (if any) would be ignored that way as we're not taking it into consideration and not passing it further down to `localizeUrl()`. Should we fix that?

Reply:
`localizeUrl` reads the locale from `i18n.getLocaleSlug()` if the `locale` param is not provided. I think we should remove that hidden dependency, but in another PR.

Reply:
Sounds good to me 👍